www/admin/advertiser-campaigns.php

```diff
@@ -122,22 +122,30 @@
         $oTz = $oActivateDate->tz;
         $oActivateDate->setTZbyID('UTC');
         $oActivateDate->convertTZ($oTz);
-        $aCampaign['activate']  = $oActivateDate->format($date_format);
+        $aCampaign['activate'] = $oActivateDate->format($date_format);
     }
     else {
-        $aCampaign['activate']  = '-';
+        $aCampaign['activate'] = '-';
     }
 
     if (!empty($aCampaign['expire_time'])) {
         $oExpireDate = new Date($aCampaign['expire_time']);
         $oTz = $oExpireDate->tz;
         $oExpireDate->setTZbyID('UTC');
         $oExpireDate->convertTZ($oTz);
-        $aCampaign['expire']    = $oExpireDate->format($date_format);
+        $aCampaign['expire'] = $oExpireDate->format($date_format);
     }
     else {
-        $aCampaign['expire']    = '-';
+        $aCampaign['expire'] = '-';
     }
+    
+    if (!empty($aCampaign['updated'])) {
```

Comment:
I was about to suggest moving this code (which is reused multiple times) to a util method somewhere, but I see that similar code was already there a few lines above... What do you think?

Reply:
I think that refactoring to remove the duplicate code is a good idea. I will do that, and update the PR. Thanks!